packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-icon-marquee/cashier-onboarding-icon-marquee.tsx

```diff
@@ -11,8 +11,8 @@ const CashierOnboardingIconMarquee: React.FC<TProps> = ({ icons }) => {
         <div data-testid='dt_cashier_onboarding_icon-marquee'>
             <NewsTicker speed={10} className='cashier-onboarding-icon-marquee'>
                 <div className={'cashier-onboarding-icon-marquee__container'}>
-                    {icons.map((icon, index) => (
-                        <Icon key={`${icon}${index}`} icon={icon} width={72} height={45} />
```

Comment:
@amir-deriv `icons` is just an array of strings, which does not guarantee it's unique. Please keep it as it was 🙏🏻

Reply:
Sonarcloud is complaining about this, do you know of anything else which can be used here for key other than `index` ?

Reply:
@amir-deriv Maybe we should revisit our Sonarcloud rules instead of just fixing the raise errors :)) It's totally fine to use `index` as `key` if you know what you are doing.

Reply:
Yeah, I am using `index` now